src/data-structures/linked-list/LinkedList.js

```diff
@@ -190,10 +190,17 @@ export default class LinkedList {
     return nodes;
   }
 
+  /**
+   * @param {Array} Array of nodes
+   */
+  fromArray(arr = []) {
+    arr.map(node => this.append(node.value));
```

Comment:
Please use forEach instead of `map` since you're not using `map` function output.